### PR TITLE
Add Random Quote Page

### PR DESCRIPTION
**Pull request by ofarrero**
This was created before we agreed to create tests, I will make tests for before the end of the week

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches a random quote from the server and adds it to the DOM.
+ */
+function getRandomQuote() {
+  console.log('Fetching a random quote.');
```
Comment (marvel308) [inline portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java:19]: Why do you want to log this to console?

Comment (ofarrero) [reply portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java:19]: I must have left it there from when I was debugging thank you, I will remove now

File: portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function handleResponse(response) {
+  const textPromise = response.text();
+  // When the response is converted to text, pass the result into the
+  // addQuoteToDom() function.
+  textPromise.then(addQuoteToDom);
+}
+
+/** Adds a random quote to the DOM. */
+function addQuoteToDom(quote) {
+  console.log('Adding quote to dom: ' + quote);
```
Comment (marvel308) [inline portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java:38]: There are still a lot of console.log() left in the files :)

Comment (ofarrero) [reply portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java]: Actually Done this time

**Pull request merged**